Editor/UI/JitInspectorView.cs

```diff
@@ -0,0 +1,260 @@
+                    var mtvi = GetMethodItems(t);
+
+                    var qualifiers = "<color=#569cd6>" 
+                                   + (t.IsAbstract ? "abstract " : string.Empty)
+                                   + (t.IsValueType ? "struct " : "class ")
+                                   + "</color>";
+                    var name = qualifiers + HighlightTypeName(t);
+                    TreeViewItem treeViewItem = new TreeViewItem(name, t, mtvi, mtvi.Count > 0);
+                    return treeViewItem;
+                })
+                .Where(tvi => tvi.Children.Count > 0)
+                .ToList();
+        }
+        private Type[] CacheLoadTypes(Assembly assembly)
+        {
+            if (!_assemblyTypes.ContainsKey(assembly))
+                _assemblyTypes[assembly] = assembly.GetTypes();
+
+            return _assemblyTypes[assembly];
+        }
+        private List<TreeViewItem> GetMethodItems(Type type)
+        {
+            var methods = type.GetMethods(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.Instance | BindingFlags.DeclaredOnly)
+                .Where(m => !m.IsGenericMethod && m.GetMethodBody() != null && !m.MethodImplementationFlags.HasFlag(MethodImplAttributes.InternalCall))
+                .OrderBy(m => m.Name)
+                .ToList();
+
+            return methods.Select(m => new TreeViewItem(GetMethodSignature(m), m)).ToList();
+        }
+        private string GetMethodSignature(MethodInfo method)
+        {
+            var sb = new StringBuilder();
+            sb.AppendTypeName(method.ReturnType);
+            sb.Append(" ");
+            sb.AppendColored(method.Name, "#dcdcaa");
+            sb.AppendColored("(", "#efb839");
+            var parameters = method.GetParameters();
+            for (int i = 0; i < parameters.Length; i++)
+            {
+                if (i > 0) sb.Append(", ");
+                sb.AppendTypeName(parameters[i].ParameterType);
+                sb.Append(" ");
+                sb.Append(parameters[i].Name);
+            }
+            sb.AppendColored(")", "#efb839");
+            return sb.ToString();
+        }
+        private static unsafe string GetDisassembly(MethodBase method)
+        {
+            if (!JitInspectorHelpers.TryGetJitCode(method, out var code, out var size))
+                return string.Empty;
+
+            return GetDisassembly(method, (byte*)code, size, new NasmFormatter
+            {
+                Options =
+                {
+                    FirstOperandCharIndex = 10
+                }
+            });
+        }
+        private static unsafe string GetDisassembly(MethodBase method, byte* code, int size, Formatter formatter)
+        {
+            using var text = new StringWriter();
+            JitInspectorHelpers.WriteDisassembly(method, code, size, formatter, text);
+            return text.ToString();
+        }
+
+    }
+    public static class StringBuilderExtensions
```

Comment:
This should be in a separate `StringBuilderExtensions.cs` and `internal` in the `JitInspector` namespace.